gr-uhd/grc/gen_uhd_usrp_blocks.py

```diff
@@ -177,6 +177,12 @@
         self.${'$'}{id}.set_time_unknown_pps(uhd.time_spec(0))
         ${'%'} elif sync == 'pc_clock':
         self.${'$'}{id}.set_time_now(uhd.time_spec(time.time()), uhd.ALL_MBOARDS)
+        ${'%'} elif sync == 'pc_clock_next_pps':
+        self.${'$'}{id}.set_time_next_pps(uhd.time_spec(int(time.time()) + 1.0))
```

Comment:
This does not sync to a PPS edge, so I would suggest leaving this mode out. If the next PPS comes immediately after this arrives at the device, the sync will be almost 1s off. In other words, this does the same as `pc_clock` except much worse.

Reply:
I had envisioned this mode being useful when the user wants the timestamps' integer values to be synchronized to the 1PPS input (i.e. 1PPS input triggers on timestamps 1616430761.0 rather than 1616430761.29557...). This might also be useful when setting/synchronizing the timestamps across different USRP devices that are connected to the same 1PPS input. While this can be accomplished using the `unknown_pps` mode, this mode also initializes the timestamps to 0. The proposed `pc_clock_next_pps` mode initializes the timestamps using the system's UTC epoch time, which gives the user a better sense of absolute time of each timestamp.

To address your point of sync being nearly 1s off (depending on when the timestamp is set), I have changed the code so that the time is set immediately after the 1PPS input is received. This is done using the process given in the [Device Synchronization](https://files.ettus.com/manual/page_sync.html) section of the USRP manual. Here, `get_time_last_pps()` is polled every 50 ms until a change is detected, then `set_time_next_pps()` is called.

Reply:
OK let's go with this.